data/roles.js

```diff
@@ -186,299 +193,311 @@ const roleData = {
         },
         "Sapling": {
             alignment: "Village",
             description: [
-                "Once per game can choose to turn into a tree at night.",
-                "While a tree, cannot vote and immune to most ways of dying.",
+                "Choose whether or not to grow into a tree at night.",
+                "Tree is immune to most ways of dying.",
+                "Tree cannot vote."
                 ],
         },
         "Tree": {
             alignment: "Village",
             disabled: true,
             description: [
-                "Cannot vote and immune to most ways of dying.",
+                "Tree is immune to most ways of dying.",
+                "Tree cannot vote."
                 ],
         },
         "Baker": {
             alignment: "Village",
             description: [
-                "When in the game, everyone starts with two bread.",
-                "Gives out up to two bread each night.",
-                "If all bakers die, a famine will start, killing anyone who can't eat one bread per day/night.",
+                "When baker is present in the game, all players start with two breads.",
+                "Gives out up to two breads each night.",
+                "If all bakers die, a famine will start, and each player will comsume one bread per day cycle and night cycle.",
+                "Players who run out of bread after famine has begun will die."
             ],
         },
         "Virgin": {
             alignment: "Village",
             description: [
-                "If executed by the village vote, no one will die the following night.",
+                "If executed by the village, no one will die the following night.",
             ],
         },
         "Mimic": {
             alignment: "Village",
             description: [
-                "Can choose to mimic another player at night and will change based on their alignment.",
-                "Will steal role if target was town, become villager if target was mafia, and become amnesiac otherwise.",
+                "Chooses a player at night and attempt to mimic their role.",
+                "If player is town, mimic steals their role and that player becomes a villager.",
+                "If player is mafia, mimic becomes villager.",
+                "If player is independent or monster, mimic becomes amnesiac."
             ],
         },
         "Judge": {
             alignment: "Village",
             description: [
-                "Votes weight is 2."
+                "Vote weight is worth 2 players votes in day meeting."
             ],
         },
         "Party Host": {
             alignment: "Village",
             description: [
-                "Can choose to host a party for everyone to attend once per game.",
+                "Chooses to host a party during day meeting for everyone to attend once per game on the following night.",
+                "Everyone will share a night meeting."
```

Comment:
Maybe add info on night meeting?
Something like "Everyone will share a meeting for the party at night." --Opinion notes though.